RELEASING/README.md

```diff
@@ -141,16 +250,29 @@ Example:
 python changelog.py --previous_version 0.37 --current_version 0.38 changelog --access_token {GITHUB_TOKEN} --risk
 ```
 
-The script will checkout both branches and compare all the PR's, copy the output and paste it on the `CHANGELOG.md`
+The script will checkout both branches and compare all the PR's and output the line that are needed to be added to the
+`CHANGELOG.md` file in the root of the repo. Remember to also make sure to update the branch id (with the above command
+`1.5` needs to be changed to `1.5.0`)
 
 Then, in `UPDATING.md`, a file that contains a list of notifications around
 deprecations and upgrading-related topics,
 make sure to move the content now under the `Next Version` section under a new
 section for the new release.
 
+#### Patch release (e.g. 1.5.1)
+
+To compare the forthcoming patch release with the latest release from the same branch, et the previous
```

Comment:
"et the previous version"?

![et-o-extraterrestre-steven-spielberg-reproducao_widelg](https://user-images.githubusercontent.com/70410625/170694025-fbc88900-e882-4b05-8172-ee0407684816.jpeg)

Reply:
🤣 🤣 🤣

Reply:
et the final version 😢 
![image](https://user-images.githubusercontent.com/33317356/170695795-b456aab7-9258-4d79-8f5a-a5bb70ea46fe.png)

Reply:
😆 😆 😆